File: app/src/main/java/com/omar/retromp3recorder/app/di/RecorderDefaults.java
package com.omar.retromp3recorder.app.di;

import com.omar.retromp3recorder.app.recorder.VoiceRecorder;

public class RecorderDefaults {
    public final VoiceRecorder.BitRate bitRate;
    public final VoiceRecorder.SampleRate sampleRate;
    public final String filePath;

    public RecorderDefaults(VoiceRecorder.BitRate bitRate, VoiceRecorder.SampleRate sampleRate, String filePath) {
        this.bitRate = bitRate;
        this.sampleRate = sampleRate;
        this.filePath = filePath;
    }
}

File: app/src/main/java/com/omar/retromp3recorder/app/modules/recording/RecorderDefaults.kt
package com.omar.retromp3recorder.app.modules.recording

import com.omar.retromp3recorder.app.modules.recording.Mp3VoiceRecorder.BitRate
import com.omar.retromp3recorder.app.modules.recording.Mp3VoiceRecorder.SampleRate

data class RecorderDefaults(
    val bitRate: BitRate,
    val sampleRate: SampleRate,
    val filePath: String
)
